pdr_backend/cli/cli_module.py

```diff
@@ -346,3 +347,14 @@ def do_deploy_pred_submitter_mgr(args, nested_args=None):
 # pylint: disable=unused-argument
 def do_sim_plots(args, nested_args=None):
     sim_dash(args)
+
+
+@enforce_types
+# pylint: disable=unused-argument
+def do_arima_plots(args, nested_args=None):
+    ppss = PPSS(
+        yaml_filename=args.PPSS_FILE,
+        network="development",
+        nested_override_args=nested_args,
+    )
+    arima_dash(8050, ppss)
```

Comment:
Regarding the port related feedback I propose:
- don't make it configurable, if user really wants to use another port then it can easily be changed in the code
- webbrowser.open() was requested by Trent so let's keep it

I will clean up the port related code to reflect the things above